app/src/main/res/layout/widget_button.xml

```diff
@@ -31,18 +32,17 @@
             android:id="@+id/widgetLabelLayout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_margin="2dp"
-            android:minHeight="32sp">
+            android:layout_margin="2dp">
 
             <TextView
                 android:id="@+id/widgetLabel"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_gravity="center"
+                android:layout_gravity="bottom|center_horizontal"
                 android:text="@string/widget_label_placeholder_text_button"
                 android:textSize="12sp"
                 android:textAlignment="center"
-                android:gravity="center"
+                android:gravity="top|center_horizontal"
```

Comment:
As the TextView height is `wrap_content`, aligning it to the top doesn't make any difference so this line doesn't need to be changed.

Reply:
Thanks for mentioning